app/models/domain_setup.rb

```diff
@@ -15,7 +15,6 @@
 #
 #  index_domain_setups_on_community_id          (community_id) UNIQUE
 #  index_domain_setups_on_critical_error        (critical_error)
-#  index_domain_setups_on_domain                (domain) UNIQUE
```

Comment:
This is also related to the accidental update to db structure that should not be in this PR.